yaptide/utils/sim_utils.py

```diff
@@ -20,68 +21,13 @@ def pymchelper_output_to_json(estimators_dict: dict) -> dict:
     result_dict = {"estimators": []}
     estimator: Estimator
     for estimator_key, estimator in estimators_dict.items():
-        # est_dict contains list of pages
-        est_dict = {
-            "name": estimator_key,
-            "metadata": {},
-            "pages": []
-        }
+        filepath = dir_path / estimator_key
+        writer = JsonWriter(str(filepath), None)
```

Comment:
better approach would be to refactor JsonWriter, so it would have a method which accepts estimator and returns json dictionary.

then you could do:
```
from pymchelper.json.writer import estimator2json
est_dict = estimator2json(estimator)
```

in the `pymchelper.json.writer` module you would have:
 - module level function `estimator2json`
 - class JsonWriter which would use `estimator2json` to make the file saving.

With such design you could skip here unccessary saving of the JSON dict to the file.

Reply:
I created and issue for it https://github.com/yaptide/yaptide/issues/304
I would leave for now this way

Reply:
Good idea